docs/en/datasets/det.md

```diff
@@ -41,6 +41,11 @@ The structure of the text detection dataset directory is organized as follows.
 │   ├── imgs
 │   ├── instances_test.json
 │   └── instances_training.json
+|── ILST
```

Comment:
It seems you didn't `mkdir ILST` in the doc. Please check if the directory name is correct. Also, `ILST` or `ilst` (since I saw `ilst` on the `shared_data` path)?